server/container_create.go

```diff
@@ -393,6 +393,11 @@ func (s *Server) getAppArmorProfileName(annotations map[string]string, ctrName s
 	}
 
 	if profile == apparmor.ProfileRuntimeDefault {
```

Comment:
1. Maybe I didn't make it clear. The value of `config.ApparmorProfile` is the name of apparmor profile, when a container is requested to be started with ProfileRuntimeDefault. It's not the name of our generate profile.
But like you said, we will load ocid-default even if `config.ApparmorProfile != "ocid-default"`. User probably doesn't want this profile, since he has his own profile. So I will fix it in next commit, make it so that we only load `ocid-default` if and only if `config.ApparmorProfile == "ocid-default"`.

2. In this block, since we have `if s.appArmorProfile == apparmor.DefaultApparmorProfile` ([here](https://github.com/kubernetes-incubator/cri-o/pull/239/files#diff-dba9598e19f7903533dc9add98d410d2R397)), I think this block maybe have already met the requirement(reload `ocid-default` if and only if `config.ApparmorProfile == "ocid-default"`.)
FYI: 
```go
const (
	// DefaultApparmorProfile is the name of default apparmor profile name.
	DefaultApparmorProfile = "ocid-default"
)
```